src/nifax/control/controller/TableGralController.java

```diff
@@ -0,0 +1,179 @@
+                    v.add(category.getDescription());
+
+                    modelTable.insertRow(0, v);
+
+                });
+
+                break;
+            case Store:
+                StoreOperation storeOperation = StoreOperation.getInstance();
+                Map<String, Store> storeList = storeOperation.List();
+                MapDb.storeList = storeList;
+                storeList.entrySet().stream().map((entry) -> entry.getValue()).forEach((store) -> {
+                    DefaultTableModel modelTable = (DefaultTableModel) panelGeneralAdmin.getTbl_gral().getModel();
+                    Vector v = new Vector();
+
+                    v.add(false);
+                    v.add(store.getId());
+                    v.add(store.getDescription());
+
+                    modelTable.insertRow(0, v);
+
+                });
+
+                break;
+            case Measure:
+                MeasureOperation measureOperation = MeasureOperation.getInstance();
+                Map<String, Measure> measureList = measureOperation.List();
+                MapDb.measureList = measureList;
+                measureList.entrySet().stream().map((entry) -> entry.getValue()).forEach((measure) -> {
+                    DefaultTableModel modelTable = (DefaultTableModel) panelGeneralAdmin.getTbl_gral().getModel();
+                    Vector v = new Vector();
+
+                    v.add(false);
+                    v.add(measure.getId());
+                    v.add(measure.getDescription());
+
+                    modelTable.insertRow(0, v);
+
+                });
+                break;
+            case Price:
+                PriceOperation priceOperation = PriceOperation.getInstance();
+                Map<String, Price> priceList = priceOperation.List();
+                MapDb.priceList = priceList;
+                priceList.entrySet().stream().map((entry) -> entry.getValue()).forEach((price) -> {
+                    DefaultTableModel modelTable = (DefaultTableModel) panelGeneralAdmin.getTbl_gral().getModel();
+                    Vector v = new Vector();
+
+                    v.add(false);
+                    v.add(price.getId());
+                    v.add(price.getDescription());
+                    v.add(price.getProfit());
+
+                    modelTable.insertRow(0, v);
+
+                });
+                break;
+
+        }
+
+        panelGeneralAdmin.getTbl_gral().setAutoCreateColumnsFromModel(false);
+        DefaultTableModel model = (DefaultTableModel) panelGeneralAdmin.getTbl_gral().getModel();
+        Vector data = model.getDataVector();
+        Collections.sort(data, new ColumnSorter(1));
+        model.fireTableStructureChanged();
+
+        panelGeneralAdmin.repaint();
+
+    }
+
+    public void delete(String tab) {
+        try {
+            HQLOperation op = HQLOperation.getInstance();
+
+            int indexBoolean = this.panelGeneralAdmin.getTbl_gral().getColumnModel().getColumnIndex("");
```

Comment:
I think allways when you use a switch and if the code on some case is not very simple, you need to clarify the purpose of the case using comments or simplifying the code in one method